tools/run_tests/run_tests.py

```diff
@@ -759,8 +759,10 @@ def dockerfile_dir(self):
             self.python_manager_name(), _docker_arch_suffix(self.args.arch))
 
     def python_manager_name(self):
-        if self.args.compiler in ['python3.5', 'python3.6']:
-            return 'pyenv'
+        if self.args.compiler in [
+                'python2.7', 'python3.5', 'python3.6', 'python3.7'
+        ]:
+            return 'stretch_%s' % re.sub(r'\D*', '', self.args.compiler)
```

Comment:
how about `return 'stretch_' + self.args.compiler[len('python'):]`

Reply:
I am fine with both solution.
Just that we will have a dot in naming...

Reply:
I prefer something obvious than regex. Having dot is fine.